beacon-chain/db/kv/epbs.go

```diff
@@ -0,0 +1,15 @@
+package kv
+
+import (
+	"context"
+
+	"github.com/prysmaticlabs/prysm/v5/consensus-types/interfaces"
+)
+
+func (s *Store) SignedExecutionPayloadHeader(ctx context.Context, blockRoot [32]byte) (interfaces.ROSignedExecutionPayloadHeader, error) {
+	b, err := s.Block(ctx, blockRoot)
+	if err != nil {
+		return nil, err
+	}
+	return b.Block().Body().SignedExecutionPayloadHeader()
```

Comment:
If `blockRoot` doesnt exist in db, `b` will be nil then I think `b.Block()` will panic. 
`SignedExecutionPayloadHeader` should have unit tests